content/post/graphql-with-go-and-postgresql.md

```diff
@@ -0,0 +1,377 @@
++++
+date = "2017-01-05T21:25:28Z"
+title = "graphql with go and postgresql"
+draft = true
+
++++
+
+# GraphQL In Use: Building a Blogging Engine API with Golang and PostgreSQL
+
+## Abstract
+
+GraphQL appears hard to use in production: the graph interface is flexible in its modeling capabilities but is a poor match for relational storage, both in terms of implementation and performance.
+
+In this document, we will design and write a simple blogging engine api, with three types of resources (users, posts and comments) and a varied set of functionality (create a user, create a post, add a comment to a post, follow posts and comments from another user, etc.), use PostgreSQL as the backing datastore (chosen because it's a popular relational DB) and write the API implementation in golang (a popular language for writing APIs).
```

Comment:
Perhaps it would be easier to understand the paragraph if bullet points would be used.